src/sidebar/components/SelectionTabs.tsx

```diff
@@ -74,7 +74,7 @@ function Tab({
       <>
         {children}
         {count > 0 && !isWaitingToAnchor && (
-          <span className="relative bottom-[3px] left-[2px] text-tiny">
+          <span className="relative bottom-[3px] left-[2px] text-[10px]">
```

Comment:
This was the sole use of `text-tiny` on actual text.

This is non-standard text: superscript count of annotations. Our goal is more that it is sized to 10px; it doesn't relate to other type sizes in the "scale."